docs/tutorials/coord_sys_tutorial.md

```diff
@@ -0,0 +1,211 @@
+# Coordinate System Tutorial
+
+## Overview
+
+MMDetection3D uses three different coordinate systems. The existence of different coordinate systems in the society of 3D object detection is necessary, because for various 3D data collection devices, such as LiDAR, depth camera, etc., the coordinate systems are not consistent, and different 3D datasets also follow different data formats. Early works, such as SECOND, VoteNet, convert the raw data to another format, forming conventions that some later works also follow, making the conversion between coordinate systems even more complicated.
```

Comment:
I think `LiDAR, depth camera, etc.` should be `LiDAR, depth, camera, etc.` because there are three types of coord systems.